### PR TITLE
Docs: draft the dynamic modeling online service document

### PR DESCRIPTION
**Pull request by kevin-y-wang**

### CODE REVIEW

File: docs/specs/dynamic_modeling/dynamic_modeling.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# Vehicle Dynamic Modeling
+
+Vehicle dynamic modeling service employs the supervised machine learning algorithm to generate a learning-based vehicle dynamic model, which can be used as the customized dynamic model in the Apollo simulation platform for control-in-the-loop simulation. The usage of the dynamic modeling asks for three main steps: (1) collect the training/test data for learning-based modeling, via a frontend data collection process monitoring system, (2) upload the collected data into cloud by locally build the standard BOS-linked folder, and (3) submit the service command via the online service webpage and expect the modeling results within email notice.
+
+<!-- # Table of Contents 1\. [Frontend](#frontend) 2\. [Data](#data) - [Upload Tool](#upload) - [Download Tool](#download) 3\. [Visulization](#visulization) -->
+
+ ## Frontend
```
Comment (Capri2014) [inline docs/specs/dynamic_modeling/dynamic_modeling.md:7]: indentation wrong

File: docs/specs/dynamic_modeling/dynamic_modeling.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
+# Vehicle Dynamic Modeling
+
+Vehicle dynamic modeling service employs the supervised machine learning algorithm to generate a learning-based vehicle dynamic model, which can be used as the customized dynamic model in the Apollo simulation platform for control-in-the-loop simulation. The usage of the dynamic modeling asks for three main steps: (1) collect the training/test data for learning-based modeling, via a frontend data collection process monitoring system, (2) upload the collected data into cloud by locally build the standard BOS-linked folder, and (3) submit the service command via the online service webpage and expect the modeling results within email notice.
```
Comment (Capri2014) [inline docs/specs/dynamic_modeling/dynamic_modeling.md:3]: Add our new entrance point as well, `bce.apollo.auto` for the fuel service.

File: docs/specs/dynamic_modeling/dynamic_modeling.md (hunk 1 above)
Comment (kevin-y-wang) [reply docs/specs/dynamic_modeling/dynamic_modeling.md]: fixed.

File: docs/specs/dynamic_modeling/dynamic_modeling.md (hunk 2 above)
Comment (kevin-y-wang) [reply docs/specs/dynamic_modeling/dynamic_modeling.md:3]: fixed. thanks

**Review by Capri2014 — APPROVED** [review]

**Pull request merged**